lib/main.js

```diff
@@ -0,0 +1,152 @@
+'use babel'
+
+/* @flow */
+
+import Path from 'path'
+import { CompositeDisposable } from 'atom'
+
+type Linter$Provider = Object
+
+module.exports = {
+  config: {
+    executablePath: {
+      type: 'string',
+      default: Path.join(__dirname, '..', 'node_modules', 'jshint', 'bin', 'jshint'),
+      description: 'Path of the `jshint` executable.'
+    },
+    lintInlineJavaScript: {
+      type: 'boolean',
+      default: false,
+      description: 'Lint JavaScript inside `<script>` blocks in HTML or PHP files.'
+    },
+    disableWhenNoJshintrcFileInPath: {
+      type: 'boolean',
+      default: false,
+      description: 'Disable linter when no `.jshintrc` is found in project.'
+    },
+    lintJSXFiles: {
+      title: 'Lint JSX Files',
+      type: 'boolean',
+      default: false
+    }
+  },
+  activate() {
```

Comment:
Not a huge deal, but it's nicer to have the extra separation of a blank line between major sections like this.